src/main/webapp/build.txt

```diff
@@ -0,0 +1 @@
+This is a page with out build info
```

Comment:
If you keep this text, "with out" should be "without".  

However, you need to consider who may see this text.  The users and operations people should never see it if the build is 100% successful and the script for replacing this file is correct.  They will see this text if there is a problem with the build or the script, though.  Therefore, the default text should be somewhat informative about the situation.

For example, see src/main/webapps/build.txt in GDrive for another approach.